src/components/CommentInfo/CommentInfo.jsx

```diff
@@ -2,0 +1,17 @@
+import './CommentInfo.scss';
+
+export const CommentInfo = ({ comment }) => (
+  <div className="CommentInfo">
+    <div className="CommentInfo__title">
+      <strong className="CommentInfo__name">{comment.name}</strong>
+
+      {' by '}
+
+      <a className="CommentInfo__email" href={`mailto:${comment.email}`}>
```

Comment:
Ensure that the `comment.email` value is sanitized or validated to prevent potential security issues, such as injection attacks, when used in the `href` attribute.